config/api_platform/resources.yaml

```
@@ -1,4 +1,14 @@
 resources:
+  App\Customer\Domain\Entity\Customer:
+    normalizationContext:
+      groups: [ 'output' ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)